lib/redux/slices/testimonials.ts

```diff
@@ -16,8 +18,15 @@ export const testimonialsSlice = createSlice({
     reducers: {
         setTestimonials: (state, { payload }) => {
             state.reviews = payload
+        },
+        setDisplay: (state, { payload }) => {
+            const index = state.reviews.findIndex(
```

Comment:
Sorry but you'll need to explain this one to me. What are we doing here exactly?

Reply:
im getting the index using the ID in order to target the correct testimonial in the array and change that to the data sent in